packages/evm/src/utils/assetTransferHelpers.ts

```diff
@@ -104,13 +104,11 @@ export function createFungibleDepositData(depositParams: FungbileDepositParams):
   const zeroPaddedAddrLen = hexZeroPad(addressLenInHex, HEX_PADDING);
   let depositData = concat([zeroPaddedAmount, zeroPaddedAddrLen, recipientAddressSerialized]);
 
-  if (optionalGas) {
+  if (optionalMessage !== undefined && optionalGas !== undefined) {
```

Comment:
what's the type of this two? numbers or undefined?

Reply:
`BigInt` and `string`